T/TDLib/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "tdlib"
```

Comment:
This _must_ match exactly the directory name, including the case
```suggestion
name = "TDLib"
```